### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by Franko1307**
update README
  add logo
  add screencast to README
  update links

addresses https://github.com/systemslab/popper/issues/393

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# Popper
+<img src="docs/figures/popper_logo.png" width="200">
```
Comment (ankan17) [inline README.md]: This can be changed to the markdown syntax instead of mixing html and markdown.

File: README.md [hunk 2]
```diff
@@ -14,12 +16,15 @@ Popper is a convention and CLI tool for generating reproducible papers. This rep
 
 Quick links to other resources:
 
-  * [5-minute screencast demo of the CLI](https://asciinema.org/a/xzEYfI3U4H5CWdmjBe0O2CAw8).
   * [Slidedeck introducing the convention](https://www.slideshare.net/ivotron/the-popper-experimentation-protocol-and-cli-tool-86987253).
   * [10-min video recording of intro and demo](https://air.mozilla.org/mozilla-open-leaders-round-4-final-demos-open-succulent-project/#@52m0s).
   * [Software Carpentry formatted Lesson](https://popperized.github.io/swc-lesson/).
   * [List of repositories that follow the convention](https://github.com/popperized).
 
+#### 5 minute screencast demo of the CLI
```
Comment (ankan17) [inline README.md]: I think it will be better if the screencast can be moved above the quick links section.

File: README.md (hunk 1 above)
Comment (Franko1307) [reply README.md]: Yeah sure, the problem with the Markdown syntax is that it doesn't support a size parameter on GitHub. Take a look of how the logo looks like and if you're okay with the size I'll use that syntax. Or if you know a way to change the size using the Markdown syntax please let me know.

Comment (ankan17) [reply README.md]: This is simple. You can use a resized logo instead. You can just resize it and add a `small_logo.png` or something like that and then use it.

Comment (Franko1307) [reply README.md]: I guess you're right. I made the changes :smile:

File: README.md [hunk 3]
```diff
@@ -12,14 +11,20 @@ Popper is a convention and CLI tool for generating reproducible papers. This rep
     convention and the CLI tool.
   * Contents of our [landing page](http://falsifiable.us) ([`gh-pages`](https://github.com/systemslab/popper/tree/gh-pages) branch).
 
+
+#### 5 minute screencast demo of the CLI
+  
+  [![asciicast](https://asciinema.org/a/xzEYfI3U4H5CWdmjBe0O2CAw8.png)](https://asciinema.org/a/xzEYfI3U4H5CWdmjBe0O2CAw8)
```
Comment (UjjwalAyyangar) [inline README.md]: Can the size of the image be reduced?

Comment (ivotron) [reply README.md]: It might be better to leave it out as we had it before. Or to reduce the size of it and put it to the end.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,8 @@
-# Popper
-
+![popper logo](docs/figures/rsz_popper_logo.png)
+#
```
Comment (ivotron) [inline README.md]: Can we reduce the size of the image and/or center it? Also, having the badges next to the title of the project (like the [hashicorp](https://github.com/hashicorp) projects), and then the logo something like:

# Popper [Build Status](https://travis-ci.org/systemslab/popper) [Join the chat at https://gitter.im/systemslab/popper](https://gitter.im/falsifiable-us/popper?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)  [PyPI version](https://badge.fury.io/py/popper) [GitHub license](https://github.com/systemslab/popper/blob/master/LICENSE)[Popper Status](http://popper.rtfd.io/en/latest/sections/badge_server.html)

<p align="center" style="text-align:center;">
  
</p>

Popper is a convention and CLI tool for generating reproducible papers...

Comment (Franko1307) [reply README.md]: It should be okay now @ivotron

**Pull request merged**